roles/container-engine/docker/templates/docker-options.conf.j2

```diff
@@ -1,11 +1,14 @@
 [Service]
 Environment="DOCKER_OPTS={{ docker_options|default('') }} --iptables={{ docker_iptables_enabled | default('false') }} \
 --exec-opt native.cgroupdriver={{ docker_cgroup_driver }} \
-{% for i in docker_insecure_registries %}--insecure-registry={{ i }} {% endfor %} \
-{% for i in docker_registry_mirrors %}--registry-mirror={{ i }} {% endfor %} \
+{% for i in docker_insecure_registries %}--insecure-registry={{ i }} \{% endfor %}
+
+{% for i in docker_registry_mirrors %}--registry-mirror={{ i }} \{% endfor %}
+
```

Comment:
With this approach, the code looks not much better than it did before, also I have some suspicions about using multiple entries in the `docker_registry_mirrors`. The way the code worked before is that it put all related parameters (i.e. `--registry-mirror=X`) in a single line, the output was functional even though inelegant when certain parameters were empty and the code was easy to look at.

With this change, it looks to me like the code is more difficult to maintain.